scripts/sentry-watcher.sh

```
@@ -0,0 +1,208 @@
+fi
+
+# ─── Fetch All Projects ──────────────────────────────────────────────────────
+log "Fetching projects for org: $ORG"
+PROJECTS_JSON=$(api_get "/organizations/$ORG/projects/")
+PROJECT_SLUGS=$(echo "$PROJECTS_JSON" | jq -r '.[].slug')
+PROJECT_COUNT=$(echo "$PROJECTS_JSON" | jq 'length')
+log "Found $PROJECT_COUNT projects"
+
+# ─── Scan All Projects ───────────────────────────────────────────────────────
+ALL_ISSUES="[]"
+NEW_ISSUES="[]"
```

Shellcheck (0.11.0)

[warning] NEW_ISSUES appears unused. Verify use (or export if used externally).

(SC2034)